src/test/java/org/fundacionjala/coding/dennis/TestCaseRental.java

```diff
@@ -0,0 +1,186 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class TestCaseRental {
+
+    public static final double LIMIT = 0.01;
+    public static final int DAYSRENTED = 3;
+    public static final int DAYSRENTED4 = 4;
+
+    /**
+     *
+     */
+    @Test
+    public void testPriceMovieRentalAtmPass() {
+        Rental rent = new Rental(new MovieNewRelease("Isle of dogs"), DAYSRENTED);
+        final float expected = 9.0f;
+        assertEquals(expected, rent.calculatePriceMovie(), LIMIT);
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testFrequentRenterPointsAtmPass() {
+        Rental rent = new Rental(new MovieNewRelease("Infinity War"), DAYSRENTED);
+        int expected = 2;
+        assertEquals(expected, rent.calculateFrequentRenterPoints());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testGetPriceCodeMoviePass() {
+        Movie mov = new MovieNewRelease("Annihilation");
+        final float expected = 3.0f;
+        assertEquals(expected, mov.getPriceCode().getValue(), LIMIT);
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testSetPriceCodeMoviePass() {
+        Movie mov = new MovieNewRelease("Annihilation");
+        mov.setpriceCode(PriceCode.REGULAR);
+        final float expected = 2.0f;
+        assertEquals(expected, mov.getPriceCode().getValue(), LIMIT);
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testGetTitleMoviePass() {
+        Movie mov = new MovieNewRelease("Annihilation");
+        String expected = "Annihilation";
+        assertEquals(expected, mov.getTitle());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testReturnFrequentPointsMoviePass() {
+        Movie mov = new MovieChildren("Ice Age");
+        int expected = 1;
+        assertEquals(expected, mov.returnFrequentPoints(DAYSRENTED));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testAddRentalCustomerPass() {
+        final int daysRented = 7;
+        Customer c = new Customer("Tester");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Tester" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)